docs/guide/extending/extending_thumbnails.md

```diff
@@ -0,0 +1,127 @@
+# Extending thumbnails
+
+Thumbnails allow you to easily choose an image for a specific Content Type.
```

Comment:
```suggestion
Thumbnails enable you to easily choose an image for a specific Content Type.
```
Add a sentence on where the thumbnail is used (content browse? sub-items?), and what thumbnails are used when you don't customize anything